docs/docs/components/pressable.mdx

```diff
@@ -0,0 +1,162 @@
+---
+id: pressable
+title: Pressable
+sidebar_label: Pressable
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import GifGallery from '@site/components/GifGallery';
+
+<GifGallery>
+  <img src={useBaseUrl('gifs/pressable.gif')} width="70%" />
+</GifGallery>
+
+:::info
+This component is a drop-in replacement for the `Pressable` component, rewritten using `Gesture Handler`.
+:::
```

Comment:
Do we need that information? And if we do, I don't think we have to say that it was rewritten using Gesture Handler - it should be pretty obvious.

```suggestion
:::info
This component is a drop-in replacement for the `Pressable` component.
:::
```

Reply:
With this one I tried to follow `Reanimated Swipeable` documentation which features almost identical info box:
https://github.com/software-mansion/react-native-gesture-handler/blob/673e037b977fa586ac3ada235ee501b91b4e2320/docs/docs/components/reanimated_swipeable.md?plain=1#L15